product_brand/tests/test_product_brand.py

```diff
@@ -5,6 +5,20 @@
 
 
 class TestProductBrand(CommonCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
You have already put this in the base test. No need to put it again.

Reply:
Okay, thanks for the correction. I have already made the changes